addons/briefing/XEH_preInit.sqf

```diff
@@ -6,4 +6,7 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
+GVAR(brief_addCredits) = false;
```

Comment:
I don't think this is safe for jips as it will overwrite the `setVariable [,,true]`

Reply:
You're right, I'll rebuild the references with getVariable where if it's missing it defaults to false.